### DISCUSSION TITLE
Trailforks integration

### DISCUSSION THREAD

**Discussion by Big-Rob-Page** [SauceLLC/sauce4strava#196] [General]
I am having issues with trailforks integration, I ticked the box for it, but the extension is still set as "no access needed" when I am on any website except strava. There is a place where (according to directions I have found) you should be able to add a site, but there is no button. Setting the extension to work on all sites is also not available.   I could not find any other tickets, I must be the only person trying to do this :(

**Comment by mayfield** [comment]
Hi @Big-Rob-Page,  Trailforks integration just means information from trailforks will be integrated into the strava page when applicable.  Essentially on the segments listing it will try to get trail report info.  It won't alter or interact with trailforks.com itself so there is no need to alter the list of sites that the ext is used.

**Comment by Big-Rob-Page** [comment]
Thanks for getting back to me, it works 180 degrees differently than I thought. I am not going to be using Strava anymore and wanted a similar tool for trail forks.

Big-Rob
